iommi/form.py

```diff
@@ -181,6 +181,8 @@ def create_or_edit_object__post_handler(*, form, is_create, **_):
     if is_create:
         assert form.instance is None
         form.instance = evaluate(form.extra.new_instance, **form.iommi_evaluate_parameters())
+        # noinspection PyProtectedMember
+        form._evaluate_parameters['instance'] = form.instance
```

Comment:
maybe a comment on why not own_evaluate_parameters is used here?

Reply:
I'll add one post merge.